config/database.yml

```diff
@@ -14,7 +14,7 @@ default: &default
   encoding: utf8mb4
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   username: root
-  password:
+  password: <%= ENV["MYSQL_ROOT_PASSWORD"] %>
```

Comment:
Would prefer to use `url` attribute instead of username + password to limit the environment variables. Also, since Heroku will provide us with a url, it would be easier to use that. Can you remove `username`, `password` and replace with `url`? They show an example on line 39.

Reply:
Done. Check it over when you can.